src/domain/i18n/locales/sv.json

```
@@ -75,11 +75,12 @@
     "TEMPERATURE": "Temperatur",
     "LINKS": "Länkar",
     "GET_ROUTE": "Väg hit",
+    "EXTRA_INFO": "Se information om badvattenkvalitet",
     "NOTICE": "Meddelande",
     "WATER_TEMPERATURE": "Vattnets temperatur",
     "WATER_TEMPERATURE_SENSOR": "Vattentemperatur sensor",
     "CLOSE": "Stänga",
-    "SEE_ON_SERVICE_MAP": "Se på Palvelukartta"
+    "SEE_ON_SERVICE_MAP": "Mer information om servicekarta"
```

I would say that this translation is a bad one: This is like "Get more information about servicemap". I'm not a Swedish speaking guy, but I would say "Mer information på servicekartan" would be better.

```suggestion
    "SEE_ON_SERVICE_MAP": "Mer information på Servicekartan"
```

---

@nikomakela Good point, swedish translation updated now